code/game/machinery/abominationmachine.dm

```diff
@@ -0,0 +1,204 @@
+////////////////////////////////////////
+//Monster beacon
+////////////////////////////////////////
+
+/obj/item/sbeacondrop/abomachine
+	desc = "A label on it reads: <i>Warning: Activating this device will send a power draining device to your location</i>."
+	droptype = /obj/machinery/abomachine
+	
+////////////////////////////////////////
+//Actual Machine
+////////////////////////////////////////
+	
+/obj/machinery/abomachine
+	icon = 'icons/obj/machines/cloning.dmi'
+	icon_state = "pod_g"
+	name = "syndicate bomb"
+	desc = "A large and menacing device. Can be bolted down with a wrench."
+
+	anchored = FALSE
+	density = FALSE
+	layer = BELOW_MOB_LAYER //so people can't hide it and it's REALLY OBVIOUS
+	resistance_flags = FIRE_PROOF | ACID_PROOF
+	speed_process = TRUE
+	interaction_flags_machine = INTERACT_MACHINE_OFFLINE
+
+	var/has_creature = FALSE
+	var/n_arms_eaten = 0
+	var/n_legs_eaten = 0
+	var/n_organs_eaten = 0
+	var/n_ublood_eaten = 0
+
+/obj/machinery/abomachine/proc/release_creature(premature = FALSE)
+	var/mob/living/simple_animal/hostile/netherworld/blankbody/creature = new (src.loc)	//to be replaced? Blank bodies are cool too
+	
+	creature.name = "abomination"
+	creature.desc = "a mass of limbs and organs disgustingly welded together with flesh."
+	
+	//life	
+	creature.maxHealth = 40+max(n_organs_eaten,20)*15
+	creature.health = creature.maxHealth
+	if (premature)
+		creature.health = creature.maxHealth/2
+	
+	//Limbs
+	creature.speed = 5-max(n_legs_eaten,10)/2
+	creature.melee_damage = 10+5*max(n_arms_eaten,10)
+	creature.obj_damage = 10
+	if (n_arms_eaten>=4)
+		creature.obj_damage = (n_arms_eaten>=8) ? 100 : 40
+	
+	to_chat(user, "<span class='warning'>It lives!</span>")
+	
+	//Reset the machine
+	has_creature = FALSE
+	n_arms_eaten = 0
+	n_legs_eaten = 0
+	n_organs_eaten = 0
+	n_ublood_eaten = 0
+	update_icon()
+	
+	return creature
+
+/obj/machinery/abomachine/obj_break()
+	release_creature(TRUE)
+	..()
+
+/obj/machinery/abomachine/obj_destruction()	//what is the difference?
+	release_creature(TRUE)
+	..()
```

Comment:
`obj_break` is called when an item reaches it's failure integrity, `obj_integrity_failure`
`obj_destruction` is called when the item item is *completely destroyed*, `obj_integrity = 0`

Reply:
Thanks. I think release creature should be called on destruction?